docs/content/users/extend/custom-compose-files.md

```diff
@@ -76,3 +76,61 @@ When defining additional services for your project, we recommended following the
 [`ddev exec`](../usage/commands.md#exec), [`ddev ssh`](../usage/commands.md#ssh), and [`ddev logs`](../usage/commands.md#logs) interact with containers on an individual basis.
 
 By default, these commands interact with the `web` container for a project. All of these commands, however, provide a `--service` or `-s` flag allowing you to specify the service name of the container to interact with. For example, if you added a service to provide Apache Solr, and the service was named `solr`, you would be able to run `ddev logs --service solr` to retrieve the Solr container’s logs.
+
+## Third Party Services May Need To Trust `ddev-webserver`
+
+In some cases, a third-party service (`docker-compose.*.yaml`) may actually consume content from the `ddev-webserver` container. For example, a PDF generator like `gotenberg` might need to consume webserver content to create a PDF, or a testing service may need to do the same.
+
+In this case, the third-party service is by default not configured to trust the certificate authority used by DDEV (via `mkcert`) so you either have to use just HTTP between the two containers, or you have to make the third-party service trust the `mkcert` certificate authority.
```

Comment:
Whittle down slightly, avoid the parenthetical to improve flow, and describe _all_ the options that are about to be explained—which includes both trusting and _ignoring_ SSL issues:

```suggestion
A third-party service is not configured to trust DDEV’s `mkcert` certificate authority by default, so in cases like this you have to either use HTTP between the two containers, or make the third-party service ignore or trust the certificate authority.
```